Ignore invalid links and mail

mre opened the pull request:
Ignore Github links in the "Your Connection deserves a name" article
These are dummy links for demonstration purposes.
Also ignore email-addresses

andygrunwald commented:
Thank you. The changes looks great.
I will Merge and deploy them tomorrow morning. Thanks for taking the time
and I hope you enjoyed the article

Am Dienstag, 25. Oktober 2016 schrieb John Mercouris :

> I've made some issues to some grammar/readability, I hope they are
> helpful!
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/andygrunwald/andygrunwald.github.io/pull/4
> Commit Summary
>
>    - Capitalized Germany
>    - Stylistic changes to grammar
>    - Grammar and readability changes
>    - Sentence change to present progressive
>
> File Changes
>
>    - *M* content/post/lesson-learned-from-running-a-meetup.md
>    <https://github.com/andygrunwald/andygrunwald.github.io/pull/4/files#diff-0>
>    (30)
>
> Patch Links:
>
>    - https://github.com/andygrunwald/andygrunwald.github.io/pull/4.patch
>    - https://github.com/andygrunwald/andygrunwald.github.io/pull/4.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/andygrunwald/andygrunwald.github.io/pull/4>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AATiQI7K0bxtkry78hLN-BQ5Ny11r5f1ks5q3nPTgaJpZM4KggO4>
> .
>

jmercouris commented:
Sounds good! It was a good article indeed!
> On Oct 25, 2016, at 23:54, Andy Grunwald <notifications@github.com> wrote:
> 
> Thank you. The changes looks great.
> I will Merge and deploy them tomorrow morning. Thanks for taking the time
> and I hope you enjoyed the article
> 
> Am Dienstag, 25. Oktober 2016 schrieb John Mercouris :
> 
> > I've made some issues to some grammar/readability, I hope they are
> > helpful!
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/andygrunwald/andygrunwald.github.io/pull/4
> > Commit Summary
> >
> > - Capitalized Germany
> > - Stylistic changes to grammar
> > - Grammar and readability changes
> > - Sentence change to present progressive
> >
> > File Changes
> >
> > - *M* content/post/lesson-learned-from-running-a-meetup.md
> > <https://github.com/andygrunwald/andygrunwald.github.io/pull/4/files#diff-0>
> > (30)
> >
> > Patch Links:
> >
> > - https://github.com/andygrunwald/andygrunwald.github.io/pull/4.patch
> > - https://github.com/andygrunwald/andygrunwald.github.io/pull/4.diff
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/andygrunwald/andygrunwald.github.io/pull/4>, or mute
> > the thread
> > <https://github.com/notifications/unsubscribe-auth/AATiQI7K0bxtkry78hLN-BQ5Ny11r5f1ks5q3nPTgaJpZM4KggO4>
> > .
> >
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/andygrunwald/andygrunwald.github.io/pull/4#issuecomment-256187492>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABnQDlM94S9-m9aFnq_gwM08Tf_8Mygsks5q3nqLgaJpZM4KggO4>.
>

andygrunwald commented:
Merged and live. Thank you!

mre committed:
Ignore invalid links and mail

 Ignore Github links in the "Your Connection deserves a name" article
These are dummy links for demonstration purposes.
Also ignore email-addresses

mre committed:
Update links.yml
